core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md

```diff
@@ -0,0 +1,23 @@
+---
+description: >-
+  The Appsmith platform allows us to share data across pages either via query params or via local storage.
```

Comment:
replace "either via..." with "using URL query params & the browser local storage"

Reply:
Done